aten/src/ATen/native/sparse/cuda/SparseCUDATensorMath.cu

```diff
@@ -541,6 +542,11 @@ Tensor _sparse_sum_backward_cuda(const Tensor& grad_, const SparseTensor& input_
   TORCH_CHECK(grad_.is_cuda(), "_sparse_sum_backward_cuda: expected 'grad_' to be CUDA tensor, but got CPU tensor");
   TORCH_CHECK(input_.is_cuda(), "_sparse_sum_backward_cuda: expected 'input_' to be CUDA tensor, but got CPU tensor");
 
+  // Short circuit if grad is either zero or empty
+  if ((grad_.layout() != kStrided && !grad_._nnz()) || !grad_.numel()) {
```

Comment:
Same nit as above: `grad_.layout() != kStrided` does not imply that it will be safe to call `nnz()` method.